Automattic/jetpack
# Lazy Images: Add placeholder support

#### Changes proposed in this Pull Request:

Add placeholder support for loading lazy images.

#### Testing instructions:

In short, add images in various places and ensure that a grey placeholder initially loads, followed by the image being loaded. This process should have minimal jank.

If you're on a fast connection, you may want to consider slowing the connection down a bit so that you can actually see the placeholders. You can do this in Chrome:

https://developers.google.com/web/tools/chrome-devtools/network-performance/network-conditions

Some specific places to test:

- On a WooCommerce site with Storefront
    - Home page
    - Shop page
    - Individual product page
    - Cart page after updating quantity
- Blog listing
- Individual posts/pages

#### Proposed changelog entry for your changes:
Adds placeholder support for Lazy Images

### After

![after_placeholders_kareless_shop mov](https://user-images.githubusercontent.com/1126811/44922797-02069a00-ad0c-11e8-92ad-0a9ab8442667.gif)


### Before

![before_placeholders_kareless_shop mov](https://user-images.githubusercontent.com/1126811/44922324-c0292400-ad0a-11e8-84bb-0538d5ba8ecf.gif)
